### ISSUE TITLE
Let toml data read into a class instance

### ISSUE THREAD

**Issue by dannypsnl** [dannypsnl/cdog#8]
So we can focus on data, not exception

**Comment by dannypsnl** [comment]
By this, we can solve #7 better

**Comment by dannypsnl** [comment]
Implementing at commit: dd8fd89cdc5fcdb932315dd7f6329d536fce4d7c

**Comment by dannypsnl** [comment]
Maybe should impl like:
```c++
class Configs {
  toml::Data c;
public:
  Configs(const ifstream &f)
    :c{f} {}
  string getConfig(string target, bool required) {
    try {
      auto res = toml::get<toml::String>(this->c.get(target));
      return res;
    } catch (toml::exception e) {
      if (required) {
        cout << "Error message" << '\n';
        std::exit(1);
      }
      return "";
    }
  }
};
```

**Comment by dannypsnl** [comment]
fix at commit: 81c277d6c7a283a457a000e5e0be8e2bc5847dfe
